app/pages/index.vue

```diff
@@ -42,8 +42,8 @@ useSeoMeta({
   <main class="grow w-full lg:w-2/3 mx-auto mt-20 p-2">
     <SiteHeader />
     <h3 class="text-lg leading-snug text-zinc-600 font-bold mt-8 mb-2">
-      Open Positions
-      <span class="text-sm text-zinc-500 before:content-['('] after:content-[')']">>{{ totalPositions }}</span>
+      Open Applicants
```

Comment:
Why this?

Reply:
To add "()"  around displaying the number through css instead of text as per @shivam-sharma7 comment.

Reply:
I asked for `Open Applicants`. Why that? we want open postings only.

Reply:
Shivam was showing the format in which you have to display number of postings.